content/docs/guides/crossguard/core-concepts.md

```diff
@@ -0,0 +1,96 @@
+    validateResource: validateTypedResource(aws.s3.Bucket, (bucket, args, reportViolation) => {
+        if (bucket.acl === "public-read" || bucket.acl === "public-read-write") {
+            reportViolation("S3 buckets cannot have an ACL set as public-read or public-read-write.");
+        }
+    }),
+};
+```
+
+If you have multiple resources that require a similar policy, you can group them together under one policy. This is possible by setting `validateResource` to an array of `ResourceValidationPolicy`. The example below shows a case where we want to run similar checks against multiple types of resources.
+
+```typescript
+const elbLoggingPolicy: ResourceValidationPolicy = {
+    name: "elb-logging-enabled",
+    description: "Checks whether the Application Load Balancers and the Classic Load Balancers have logging enabled.",
+    enforcementLevel: "mandatory",
+    validateResource: [
+        validateTypedResource(aws.elasticloadbalancing.LoadBalancer, (loadBalancer, args, reportViolation) => {
+            if (loadBalancer.accessLogs === undefined || !loadBalancer.accessLogs.enabled) {
+                reportViolation("Elastic Load Balancer must have access logs enabled.");
+            }
+        }),
+        validateTypedResource(aws.elasticloadbalancingv2.LoadBalancer, (loadBalancer, args, reportViolation) => {
+            if (loadBalancer.accessLogs === undefined || !loadBalancer.accessLogs.enabled) {
+                reportViolation("Elastic Load Balancer must have access logs enabled.");
+            }
+        }),
+        validateTypedResource(aws.applicationloadbalancing.LoadBalancer, (loadBalancer, args, reportViolation) => {
+            if (loadBalancer.accessLogs === undefined || !loadBalancer.accessLogs.enabled) {
+                reportViolation("Application Load Balancer must have access logs enabled.");
+            }
+        }),
+    ],
+};
+```
+
+### Stack Validation Policy
+
+Policies of `StackValidationPolicy` are run against all the resources in a stack. These policies are run are all stack resources are registered and thus *do not* block an out-of-compliance resource from being created, but do fail the `preview` or `update`. To avoid creating out-of-compliance resources, we recommend always running a `preview` command before an `update`. This allows you to write policies where one resource depends on the state or existence of another resource.
+
+## Policy Pack
+
+A Policy Pack can contain one or more policies to enforce. Packs provide a way to group together similar policies. For example, you may decide to have one pack with AWS policies and another with Kubernetes-specific policies. That being said, there are no restrictions on which policies you combine within a pack, and you should pack them however makes sense for your organization. Currently, there is a limitation of 25 policies per pack.
+
+As part of CrossGuard, organization administrators can author and publish Policy Packs to the Pulumi Console.
```

Comment:
It would be nice to have a code snippet here for creating a policy pack.

Also, we restrict "one policy pack per process". (The relationship between an NPM module and a policy pack isn't quite covered here, and something we'll want to address in a future update.)